### DISCUSSION TITLE
Help migrating my WinUI3 to uno platform

### DISCUSSION THREAD

**Discussion by Tresillo2017** [unoplatform/uno#12438] [Q&A] [answered]
This is my first time using c#, winui3 and uno platform so I'm new to this. I followed the guide about how to migrate my solution to uno platform, but when I create the uno solution I don't know where to but my files, and when I copy them, it's just throw me a lots of error.

Source code: https://github.com/tresillo2017/vectorify

> I would appreciate some advice and general knowledge that you think is helpful for me. Thanks in advance

**Comment by limeniye** [comment]
https://github.com/tresillo2017/vectorify *

**Reply by Tresillo2017** [reply]
My bad, it'd already edited

**Reply by jeromelaban** [reply, marked as answer]
You should be able to use the steps provided here: https://platform.uno/docs/articles/howto-migrate-existing-code.html. The docs mention UWP, but you should be able to use similar techniques.

**Reply by Tresillo2017** [reply]
The problem is that my solution is giving me problems with the `Vectorify.Core` folder not being detected by the `app.cs`. I've already made the migration thanks to some people on the discord server.

**Reply by jeromelaban** [reply]
For the core project, you need to reference it from this project: https://github.com/Tresillo2017/Vectorify/tree/main/Vectorify/Vectorify

**Reply by Tresillo2017** [reply]
It's already in the app.cs but it's giving me an error about missing a assembly reference
```
using Vectorify.Core.Contracts.Services; 
using Vectorify.Core.Services;
```
[Errors.txt (discord)](https://discord.com/channels/372137812037730304/550416151172087808/1112431802188054578)

That's the file with all the errors I'm getting while building it for windows. (none of the platforms works)

**Reply by jeromelaban** [reply]
Adding a project reference is also required. I do not see any to [your core project here](https://github.com/Tresillo2017/Vectorify/blob/main/Vectorify/Vectorify/Vectorify.csproj).

**Reply by Tresillo2017** [reply]
I added the reference and the error disappeared but now I'm getting lots of errors for other things. 

https://pastebin.com/raw/gvLBMqua

**Reply by jeromelaban** [reply]
Thanks. It's generally best to take a look at the error list from the error window, and start from the first ones. The error list is generally useless in these scenarios.
